bindings/python/build_defs.oss.bzl

```diff
@@ -77,6 +77,125 @@ def iree_py_test(**kwargs):
     #   imports
     py_test(legacy_create_init = False, **kwargs)
 
+def iree_py_test_suite(
+        name,
+        srcs,
+        excludes = [],
```

Comment:
Just take srcs. You can pass a glob that has the excludes

Reply:
Nice, done.